.editorconfig

```diff
@@ -99,17 +102,38 @@ dotnet_style_collection_initializer = true:suggestion
 dotnet_style_coalesce_expression = true:suggestion
 dotnet_style_null_propagation = true:suggestion
 dotnet_style_explicit_tuple_names = true:suggestion
+dotnet_style_readonly_field = true:suggestion
+dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggestion
```

Comment:
With the setting we should replace all null reference checks with "is null" to fix Roslyn analyzer issues.

Reply:
Yeah, that'd be best. Would probably be best to put that in its own PR, there's going to be quite a lot of those I reckon. 😁

Reply:
We can accept this. Fixing code should be in follow PR.